<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="USER-PC" failures="1" tests="6" name="Default test" time="100.021" errors="0" timestamp="11 Dec 2018 06:51:03 GMT">
  <testcase classname="com.accenture.ESC.HomePageTest" name="contactTabValidation" time="0.533"/>
  <testcase classname="com.accenture.ESC.HomePageTest" name="coursesButtonScenario" time="7.854"/>
  <testcase classname="com.accenture.ESC.HomePageTest" name="headerVerification" time="0.166"/>
  <testcase classname="com.accenture.ESC.HomePageTest" name="homeButtonScenario" time="11.689"/>
  <testcase classname="com.accenture.ESC.HomePageTest" name="loginScenario" time="26.0">
    <failure type="java.lang.AssertionError" message="Wrong Credentials expected [http://www.qaclickacademy.com/] but found [https://qaclickacademy.usefedora.com/gdpr/agree_updated_privacy_policy]">
      <![CDATA[java.lang.AssertionError: Wrong Credentials expected [http://www.qaclickacademy.com/] but found [https://qaclickacademy.usefedora.com/gdpr/agree_updated_privacy_policy]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.accenture.ESC.HomePageTest.loginScenario(HomePageTest.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginScenario -->
  <testcase classname="com.accenture.ESC.HomePageTest" name="loginScenario" time="39.442"/>
</testsuite> <!-- Default test -->
